plugin.php

```diff
@@ -4,7 +4,7 @@
 $id = "xbbb";
  
 // code version; must be changed for all code changes
-$version = "1.0.14";
```

Comment:
The commit message and pull request title does not match what's done in the code.

You are actually increasing the version number.

Reply:
And actually, it's already merged, so this PR can be closed.